tests/__tests__/mixins/labels.js

```diff
@@ -1,25 +1,44 @@
 import { ATOM_GROUP_NAME } from "../../../src/enums";
 import { getWaveInstance } from "../../enums";
+import * as THREE from "three";
 
 describe("Atom labels", () => {
     let wave, atoms, labels;
 
     beforeEach(() => {
         wave = getWaveInstance();
         const atomGroup = wave.scene.getObjectByName(ATOM_GROUP_NAME);
+        labels = wave.structureGroup.children.filter((child) => child instanceof THREE.Points);
```

Comment:
Line 10 demonstrates how to get the atomGroup group by name

Reply:
Not sure that it could be correct since we can have a few atom groups with the same names.

Reply:
Let's put a note to keep the names unique in the `enums.js` or inside a constructor and use only one way to get the group - by name?